src/Microsoft.Health.ControlPlane.Core/Features/Exceptions/ControlPlaneException.cs

```diff
@@ -0,0 +1,17 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Health.ControlPlane.Core.Features.Exceptions
+{
+    public abstract class ControlPlaneException : Exception
```

Comment:
We probably should consider deriving all of our exception from MicrosoftHealthException like this one or FhirException.

Reply:
That makes sense to me. I'll update the ControlPlaneException to derive from the MicrosoftHealthException and, in the interest of containing this PR, put the FhirException derivation as an issue.